signal/signal.py

```diff
@@ -6,16 +6,25 @@
 
 SIGINT = 2
 SIGPIPE = 13
+SIGALRM = 14
 SIGTERM = 15
 
 libc = ffilib.libc()
 
 signal_i = libc.func("i", "signal", "ii")
 signal_p = libc.func("i", "signal", "ip")
+alarm_i = libc.func("I", "alarm", "I")
```

Comment:
since there is no special handling needed for arguments, this could just be

    alarm = libc.func("I", "alarm", "I")

and skip the wrapper below. The same applies to `pause`.

Reply:
Thanks. The file have been changed.